### ISSUE TITLE
`downsteam` misses descendents of ambiguous clades

### ISSUE THREAD

**Issue by arendsee** [ropensci/taxize#653]
This call

```R
taxize::downstream('Brassicaceae', db='ncbi', downto='genus')
```

loses all descendents of the clade 'Brassicaceae incertae sedis'. I believe this clade gets filtered out as ambiguous when `children` is called on 'Brassicaceae'.

In this particular case, I believe the species in 'Brassicaceae incertae sedis' are predicted to be in 'Brassicaceae', but their particular location in the family is unknown. So I think they should be included as downstream of 'Brassicaceae'.

We can pass the `ambiguous=TRUE` argument to `ncbi_downstream`. This in turn should, according to the documentation, be passed to `ncbi_children`.

```R
taxize::downstream('Brassicaceae', db='ncbi', downto='genus', ambiguous=TRUE)
```

However the results are the same, so I think the argument is not getting passed.

Also, it is reasonable that a user might want to keep ambiguous nodes but filter ambiguous species. I would suggest adding two new arguments to `ncbi_downstream`: `ambiguous_nodes=TRUE` and `ambiguous_species=FALSE`. I am not certain, though, if keeping ambiguous nodes is the right thing to do by default.

**Comment by arendsee** [comment]
I think I fixed the argument passing issue. But there is still the question of whether there should be special handling of ambiguous nodes.

**Comment by sckott** [comment]
thanks, will take a look tomorrow to familiarize myself with it, can't make educ. opinion right now (boarding 🛫 soon)

**Comment by arendsee** [comment]
No worries, have a nice flight!

**Comment by sckott** [comment]
the merge closed this, but you did say it partially addresses this, i assume you want this to remain open, yes?

**Comment by arendsee** [comment]
Yeah, there is just the matter of whether we want to be able to handle ambiguous nodes and ambiguous species differently.

**Comment by sckott** [comment]
yeah, will reopen

**Comment by sckott** [comment]
@arendsee  any further thoughts on 

> whether we want to be able to handle ambiguous nodes and ambiguous species differently.

**Comment by arendsee** [comment]
@sckott Nothing particularly new. `taxizedb` can distinguish ambiguous nodes and ambiguous species, e.g.

```R
taxizedb::downstream(3700, downto='genus', ambiguous_nodes=FALSE, ambiguous_species=TRUE)
```

Adding the same arguments to `taxize` might be good. Although I am not sure whether the break from the existing API is worth the gain in control.

**Comment by sckott** [comment]
bumping this to next milestone
